src/app/components/NavBar/NavBar.stories.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import NavBar from "./NavBar";
+
+export default {
+  title: "Component/NavBar",
+  component: NavBar,
+};
+export const NavBarContainer = (): JSX.Element => <NavBar />;
```

Comment:
It's not a `Container`. Consider choosing a different name:

```suggestion
export const Default = (): JSX.Element => <NavBar />;
```

```suggestion
export const Default = (): JSX.Element => <NavBar />;
```